### PR TITLE
Api unit tests

### PR DESCRIPTION
**Pull request by houssam-mahlous**
Added unit tests for controllers and services in the API module. Suggestions are welcome

### CODE REVIEW

File: api/src/test/java/com/brandwatch/minibcr/api/services/MentionServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        when(mentionRepositoryMock.readAllMentions()).thenReturn(new ArrayList<>());
+        assertEquals(mentionService.getMentions(), new ArrayList<>());
+    }
+
+    @Test
+    public void givenMentionsExist_thenReturnMentionsList() {
+        List<Mention> expectedMentions = new ArrayList<Mention>(Arrays.asList(
+                new Mention(), new Mention()));
+        when(mentionRepositoryMock.readAllMentions()).thenReturn(expectedMentions);
+
```
Comment (acmcmurray) [inline api/src/test/java/com/brandwatch/minibcr/api/services/MentionServiceTest.java]: Whitespace

**Review by acmcmurray — APPROVED** [review]
NABD, as the variables are only used in the tests they're declared in, but you could make them constants. Otherwise, looks good!

**Comment by houssam-mahlous** [comment]
> NABD, as the variables are only used in the tests they're declared in, but you could make them constants. Otherwise, looks good!

Will put that in mind when reusing the same variables across multiple tests.

**Review by kocevaIv — APPROVED** [review]
Looks good to me 👍

**Pull request merged**